h/services/search_index/_queue.py

```diff
@@ -199,8 +199,8 @@ def sync(self, limit):
 
         return counts
 
-    def count(self, tags=None, expired=False):
-        return self._job_query(tags=tags, expired=expired).count()
+    def count(self, tags=None):
+        return self._job_query(tags=tags).count()
```

Comment:
I've changed the expired jobs metric to count all expired jobs, not just expired `sync_annotation` jobs, and moved the query into the Celery task below instead of supporting it in this method, because it's no longer `sync_annotation`-specific

Reply:
:+1: